native/thrift_struct_read_module.c

```diff
@@ -0,0 +1,1923 @@
+#include <Python.h>
+#include <string.h>
+#include <math.h>
+#include "thrift_consts.h"
+
+
+/*some structs to handle stacks needed in TCompactProtocol and TJSONProtocol*/
+
+struct structs
```

Comment:
maybe a more descriptive name than `structs`? (which is confusing because `struct` is a reserved word...